GherkinSyncTool.Synchronizers.AzureDevOps/Client/AzureDevopsClient.cs

```diff
@@ -29,22 +31,38 @@ public AzureDevopsClient(Context context)
             _context = context;
         }
 
-        public WitBatchRequest CreateTestCaseBatchRequest(JsonPatchDocument patchDocument)
+        public WitBatchRequest BuildCreateTestCaseBatchRequest(JsonPatchDocument patchDocument)
         {
             var workItemTrackingHttpClient = _connection.GetClient<WorkItemTrackingHttpClient>();
             return workItemTrackingHttpClient.CreateWorkItemBatchRequest(_azureDevopsSettings.Project, WorkItemTypes.TestCase, patchDocument, false, false);
         }
+        
+        public WitBatchRequest BuildUpdateTestCaseBatchRequest(int id, JsonPatchDocument patchDocument)
+        {
+            var workItemTrackingHttpClient = _connection.GetClient<WorkItemTrackingHttpClient>();
+            return workItemTrackingHttpClient.CreateWorkItemBatchRequest(id, patchDocument, false, false);
+        }
+        
+        public IEnumerable<int> GetAllTestCasesIds()
+        {
+            var workItemTrackingHttpClient = _connection.GetClient<WorkItemTrackingHttpClient>();
+            var wiql = new Wiql
```

Comment:
Maybe we should use some clear names rather than abbreviations, if this is one of them?

Reply:
THe Wiql is a class Microsoft.TeamFoundation.WorkItemTracking.WebApi.Model namespace. I don't manage to rename it, but I added a comment.